lib/marshalls/provenance.marshall.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const semver = require('semver')
+// const semver = require('semver')
```

Comment:
I kept it commented out because the code that's using it is already commented out